specs/pvs/corla.pvs

```diff
@@ -863,6 +934,8 @@ BEGIN
 % @description The audited contest determines the number of ballots
```

Comment:
This situation should only affect tiny numbers of ballots and seems pretty easy to defend. If we don't assume that, we're diverging from published RLA procedures, and need an alternative in which the math works. I think we should handle it this way and cover it in the documentation for preparing manifests.